pkg/operator/controllers/machineset/machineset_controller.go

```diff
@@ -21,12 +21,12 @@ import (
 
 	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
 	aroclient "github.com/Azure/ARO-RP/pkg/operator/clientset/versioned"
-	"github.com/Azure/ARO-RP/pkg/operator/controllers"
 )
 
 const (
-	CONFIG_NAMESPACE string = "aro.machineset"
-	ENABLED          string = CONFIG_NAMESPACE + ".enabled"
+	ControllerName = "MachineSet"
+
+	ControllerEnabled = "aro.machineset.enabled"
```

Comment:
Exported becuase we use it in e2es. Other non-exported constants like this can be made exported later when/if we start using them outside of their packages.